### PR TITLE
Incorporate experimental and alpha features

### PR DESCRIPTION
**Pull request by akeller**
## Description

This PR creates an "Experimental" and "alpha" section in the Components area of the docs with a landing page, SAP, and RPA documentation.

Screenshot 2024-10-14 at 11 50 45 AM

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR) **To release Oct 15 for CCon**
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by lmbateman — COMMENTED** [review]
This seems fine to me. I can see it under Next but not 8.5, which I think is the goal.

**Comment by PhilippaC22** [comment]
@christinaausley structure looks good - Have added some design suggestions around this here as well as naming ideas too. 
[See Figjam](https://www.figma.com/board/MxP4pMBkrb0QanSDineqlB/Experimental-features?node-id=0-1&t=EyNwjxTYPvBZLTUw-1)

**Comment by akeller** [comment]
We won't use "experimental" in the official, shipped version. See my update on Slack - https://camunda.slack.com/archives/C0484UFR2VA/p1725571619654449

**Comment by camundom** [comment]
> @camundom @melanie-butcher sounds like you are perhaps putting together some design ideas? Do we have any strong opinions around some of the potential naming terms based on the Slack thread linked above? Also happy to hop on a call and discuss.

Hi @christinaausley, here's a link to the [draft design ](https://www.figma.com/design/vP5O593iF0JVtCtJ4h5tK2/Experimental-features-area?node-id=19-53150&t=UMJ621GSRVhqUUvm-4)for the docs, includes a revised structure for early access/experimental features, along with few open questions. Please let me know what you think.

**Comment by christinaausley** [comment]
Thanks for meeting with me today @camundom -- I'll let you add any changes to this PR based on conversation with @bastiankoerber, but please let me know if I can assist with anything or review when you are finished.

**Review by YanaSegal — APPROVED** [review]

**Comment by akeller** [comment]
Removing `deploy` since it's currently broken. 😢

**Comment by christinaausley** [comment]
Hi @camundom -- based on your draft design, can I go ahead and update this PR so we have a place to put content before CCon?

**Comment by christinaausley** [comment]
@akeller It's looking like I can start pulling some experimental CCon documentation into GitHub. How do we feel about merging those into this branch once the structure is ready to go?

**Comment by akeller** [comment]
> @akeller It's looking like I can start pulling some experimental CCon documentation into GitHub. How do we feel about merging those into this branch once the structure is ready to go?

Doesn't matter to me! I'll leave it with the team to decide on names, etc. Also if it's easier to close this PR out and start fresh, that works too.

**Comment by camundom** [comment]
> Hi @camundom -- based on your draft design, can I go ahead and update this PR so we have a place to put content before CCon?

Hi @christinaausley , yes that can be done. I've also received final approval to the [design/structure](https://www.figma.com/design/vP5O593iF0JVtCtJ4h5tK2/Experimental-features-area?node-id=106-87&t=XROumcKLB91lXtTM-4) that would be used for CCon.

**Comment by christinaausley** [comment]
Thanks @camundom -- prepping this today and tomorrow.

File: sidebars.js [hunk 1]
```diff
@@ -689,6 +689,21 @@ module.exports = {
         },
       ],
     },
+    {
```
Comment (christinaausley) [inline sidebars.js:692]: Will update sidebar once we get RPA and SAP in here.

**Review by PhilippaC22 — APPROVED** [review]

**Comment by christinaausley** [comment]
This is ready for final review -- I will plan to release this tomorrow afternoon. CC @bastiankoerber

File: docs/components/early-access/overview.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## What to expect
+
+|                                     | Experimental features                                                                              | Alpha features                                                                     |
+| ----------------------------------- | -------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| <b>Purpose</b>                      | Early innovation and ideation                                                                      | Test upcoming features that could make it to GA                                    |
+| <b>Suitable for production use</b>  | No                                                                                                 | No                                                                                 |
+| <b>Stability</b>                    | APIs, dependencies, and configuration are likely to change.                                        | APIs, dependencies, and                                                            |
+| configuration are likely to change. |
```
Comment (mesellings) [inline docs/components/early-access/overview.md:32]: These table rows need reformatting: 
image

File: docs/components/early-access/overview.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+You can test out these features and provide additional feedback to help Camunda understand how they perform in real-world scenarios.
+
+:::note
+These features may be unstable, and subject to significant changes or removal.
+:::
+
+## Alpha features
+
+Alpha features are more developed and closer to becoming part of the product but may not yet be fully optimized or supported. Selected Camunda features and components are released as alpha versions to provide early access. By testing these features, you have the opportunity to participate in their development by sharing feedback before they reach General Availability (GA).
```
Comment (mesellings) [inline docs/components/early-access/overview.md:23]: Do we want to link to the Alpha features page? https://docs.camunda.io/docs/next/reference/alpha-features/

**Review by mesellings — APPROVED** [review]
I've reviewed the overview page but have already reviewed the other pages - just one possibly blocking issue with the table rows, but I'm aware you want to emrge this today so approving as this can be done ina. follow-up PR 👍

File: docs/components/early-access/overview.md (hunk 3 above)
Comment (christinaausley) [reply docs/components/early-access/overview.md]: Honestly we could probably move this entire page over as an "overview" to the alpha features section, but I consider that out of scope for this PR. Will open an additional issue to revisit.

Comment (akeller) [reply docs/components/early-access/overview.md]: I was going to provide similar feedback - we include definitions or explanations for alphas and minors on the [release policy page](https://docs.camunda.io/docs/next/reference/release-policy/#alpha-features-and-releases) as well. These should be linked so they reference each other. I'm ok with this being handled in a separate PR.

Comment (christinaausley) [reply docs/components/early-access/overview.md]: Cool. I'll launch another PR. Can I get a final approval from one of you so I can go ahead and release this?

**Review by mesellings — APPROVED** [review]
Lgtm - great work! 🥇 👍

**Pull request merged**